client/src/Login/Login.js

```diff
@@ -7,78 +7,156 @@ import Button from 'react-bootstrap/Button';
+            <Container fluid>
+                <Row>
+                    <Col />
+                    <Col>
+                        <CardGroup>
+                            <Card style={{ width: '18rem' }}>
+                                <Card.Body>
+                                    <Form
+                                        noValidate
+                                        onSubmit={handleSubmit}
+                                        data-testid='login-form-validity-test'>
+                                        <Form.Group className='mb-3'>
+                                            <Form.Label htmlFor='email'>Email Address</Form.Label>
+                                            <Form.Control
+                                                type='email'
+                                                placeholder={LoginConstants.email.placeholder}
+                                                maxLength={40} id='email'
+                                                required data-testid='login-form-email-input'
+                                                onChange={(e) => onEmailChange(e)} />
+                                            <Form.Control.Feedback
+                                                type='invalid'
+                                                data-testid='email-err-msg'
+                                            >Please enter a valid email.
+                                            </Form.Control.Feedback>
+                                        </Form.Group>
+                                        <Form.Group className='mb-3'>
+                                            <Form.Label htmlFor='password'>Password</Form.Label>
+                                            <Form.Control
+                                                type='password'
+                                                placeholder='Enter Password'
+                                                minLength={LoginConstants.password.minLength}
+                                                maxLength={LoginConstants.password.maxLength}
+                                                id='password'
+                                                required
+                                                data-testid='login-form-password-input'
+                                                onChange={(e) => onPasswordChange(e)} />
+                                            <Form.Control.Feedback
+                                                type='invalid'
+                                                data-testid='password-err-msg'
+                                            >Please enter a password with a minimum of 8 chars.
+                                            </Form.Control.Feedback>
+                                        </Form.Group>
+                                        <Button onSubmit={handleSubmit} variant="primary" type="submit" disabled={!loginInfoValid}>
```

Comment:
onSubmit is not necessary since the handleSubmit is called button click as this is a submit type button associated with the form